packages/request-node/README.md

```diff
@@ -148,6 +148,10 @@ A Node can be deployed by anyone. Users interested by running their own node can
 
 ### Installation
 
+#### Install IPFS Kubo (go-ipfs)
+
+https://docs.ipfs.tech/install/command-line/#install-ipfs-kubo
```

Comment:
I recommend using docker to run IPFS

Reply:
Ah okay. I'll recommend IPFS Docker near the top of the README. But I'll still include the Kubo link lower down where it describes using `ipfs init`

Reply:
I moved the Kubo installation down to the bottom of the README where `ipfs-init` script is used.  

I don't know the best place to recommend IPFS Docker so I'm going to leave that part out for now.